FoodSuggestions/app/src/main/res/layout/activity_register.xml

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:layout_gravity="center"
+    android:scrollbars="vertical"
+    tools:context=".firebase.RegisterActivity">
+
+    <ImageView
+        android:id="@+id/imageView2"
+        android:layout_width="307dp"
+        android:layout_height="226dp"
+        android:layout_gravity="center"
+        android:background="@drawable/momandson" />
+
+    <TextView
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="Register Here"
+        android:textSize="25sp"
+        android:layout_marginStart="25dp"
+        android:textColor="#ffb366"
+        android:fontFamily="sans-serif-condensed"
+        android:layout_marginTop="5sp"
+        />
+
+    <EditText
+        android:id="@+id/idRName"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:ems="10"
+        android:hint="Full Name"
+        android:inputType="text"
+        android:maxLines="1"
+        android:layout_marginStart="25sp"
+        android:layout_marginEnd="25sp"
```

Comment:
If marginStart == marginEnd (marginTop == marginBottom), we can use marginHorizontal = 5dp || marginVertical = 5dp. The same applies also for paddings.

Reply:
Fixed